src/wasm/wasm-validator.cpp

```diff
@@ -483,6 +485,7 @@ void FunctionValidator::visitSetLocal(SetLocal *curr) {
 }
 
 void FunctionValidator::visitLoad(Load *curr) {
+  shouldBeTrue(!curr->isAtomic || info.features & Features::Atomics, curr, "Atomic operation (atomics are disabled)");
```

Comment:
i personally think it's clearer to write `if (curr->isAtomic) shouldBeTrue(..atomics feature enabled..)`

Reply:
Me too, done.